config/crd/bases/infrastructure.cluster.x-k8s.io_awsclusters.yaml

```diff
@@ -596,7 +596,9 @@ spec:
                 type: string
               sshKeyName:
                 description: SSHKeyName is the name of the ssh key to attach to the
-                  bastion host.
+                  bastion host. If nil, will use a default SSH key pair name If empty
```

Comment:
You may want to adjust your comments in the types you've adjusted in this PR so these display correctly (the generator ignores newlines)